socorro/unittest/external/es/test_connection_context.py

```diff
@@ -1,45 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-import elasticsearch
-
-from socorro.external.es.connection_context import ConnectionContext
-from socorro.unittest.external.es.base import ElasticsearchTestCase
-
-
-class IntegrationTestConnectionContext(ElasticsearchTestCase):
```

Comment:
This tested a lot of uninteresting boilerplate and also the wrapped faux connection which I rmoved, so I nixed it all.